topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.md

```diff
@@ -0,0 +1,379 @@
+human-readable summary of the clustering results, with the underlying raw data also provided. The top of the 
+report contains a summary of the settings used to run the clustering, and when it was run.
+
+> <code-out-title>MuAirss Clusters report</code-out-title>
+> ```
+> ****************************
+> |                          |
+> |       PYMUON-SUITE       |
+> |  MuAirss Clusters report |
+> |                          |
+> ****************************
+> 
+> Name: struct
+> Date: 2023-11-06 09:41:00.787599
+> Structure file(s): input_structure.cell
+> Parameter file: params.yaml
+> 
+> Clustering method: Hierarchical
+>     t = 0.4
+> 
+> *******************
+> ```
+{: .code-out}
+
+The remainder of the report details the clusters and the individual muonated 
+structures composing each cluster. In
+addition to showing the relative numbers of structures forming each cluster,
+we also report the structure with the minimum energy, the average energy for all the structures 
+and the corresponding standard deviation for that average energy. By analysing of these 
+quantities for different values of the `Poisson radius`, `Van der Waals scale` and 
+`t parameter for hierarchical clustering` parameters, we can determine which 
+are the "stable" clusters, which are the ones associated to potential stopping sites.
+
+> <code-out-title>MuAirss Clusters report</code-out-title>
+> ```
+> Clusters for struct:
+> CALCULATOR: uep
+> 	2 clusters found
+> 
+> 
+> 	-----------
+> 	Cluster 1
+> 	-----------
+> 	Structures: 7
+> 
+> 	Energy (eV):
+> 	Minimum		Average		StDev
+> 	-10.30		-10.30		0.00
+> 
+> 	Minimum energy structure: struct_7
+> 
+> 
+> 	Structure list:
+> 	struct_1	struct_3	struct_5	struct_6	
+> 	struct_7	struct_8	struct_12	
+> 
+> 	-----------
+> 	Cluster 2
+> 	-----------
+> 	Structures: 7
+> 
+> 	Energy (eV):
+> 	Minimum		Average		StDev
+> 	-11.93		-11.93		0.00
+> 
+> 	Minimum energy structure: struct_14
+> 
+> 
+> 	Structure list:
+> 	struct_2	struct_4	struct_9	struct_10	
+> 	struct_11	struct_13	struct_14	
+> 
+> 	----------
+> 
+> 	Similarity (ranked):
+> 	1 <--> 2 (distance = 1.681)
+> 
+> --------------------------
+> ```
+{: .code-out}
+ 
+The classical UEP method scans the electrostatic potential looking for minima in it, which 
+are associated to potential muon stopping sites. This method further classifies those minima 
+by estimating their "attractor size".  If you look at the picture in the figure below, the two 
+minima have similar values, but the capacity to "attract" a muon is greater for the minima on the left, 
+which makes it more likely to be a muon stopping site. The probability of a potential stopping site is 
+given by the size of the cluster that forms around its associated minima.
+
+![Classifying Mininima of the Host Material's Electrostatic Potential](../../images/theoretical_basis_of_our_variant_of_the_UEP_method.png) 
+
+
+LIMITATIONS OF THE METHOD - WARNINGS
```

Comment:
is this section pending?

Reply:
@leandro-liborio might have some text to put here - will check whether there's something to add or whether to remove this bit

Reply:
That was a reminder for myself that should be removed.

Reply:
I am planning to add concrete examples where the method works and where it doesn't work.  But it will be in a complementary tutorial